packages/fxa-settings/src/components/ErrorBoundaries/index.tsx

```diff
@@ -47,6 +50,28 @@ export class AppErrorBoundary extends React.Component<{ children: ReactNode }> {
 }
 
 export const AppError = ({ error }: { error?: Error }) => {
+  const session = useSession();
+  const navigate = useNavigate();
+
+  // Special Handling for invalid session states
+  if (
+    isAuthUiError(error) &&
+    (error.errno === AuthUiErrors.INVALID_TOKEN.errno ||
+      error.errno === AuthUiErrors.UNVERIFIED_SESSION.errno)
+  ) {
+    return (
+      <AppErrorDialog
+        errorType="invalid-session"
+        onSignInAgain={() => {
+          if (session) {
+            session.destroy();
+            navigate('/signin');
```

Comment:
I'm a little bit uncertain if this is a good idea or not. My concern is that some users might be coming from an RP, and this would change the sign in behavior. 

Maybe we should just tell them to sign in again and keep super simple for now. Clearing the current session from cache seems important to prevent a cached sign in... unless we are confident the cached sign in will handle an unverified session state properly.

Any thoughts @LZoog?

Reply:
I'm in favor of just a simple message, but unlike the banner. How does the user sign out from this application error page? I guess this is why I went to the trouble to add a button/link.